app/javascript/components/EditForm.js

```diff
@@ -94,15 +94,15 @@ class EditRebateForm extends React.Component {
                 : SingleInput({...field, isReadOnly})
               })}
             </div>
-            { values.fields.moved_within_rating_year == 'yes' &&
+            { values.fields.moved_within_rating_year == 'Yes' &&
```

Comment:
we need to have a bit of a think about this, will break previously submitted applications - need to separate business logic and presentation